src/main/resources/locales/en/case-name-search.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "Subscribe by name of the party or parties involved",
-  "heading": "What is the name of the party or parties involved?",
+  "title": "Subscribe by case name",
```

Comment:
Looks like the functional test is failing on case name. Probs needs updating

Reply:
Sorted